apps/web/src/pages/get-started/consts/EchoUseCase.const.tsx

```diff
@@ -0,0 +1,95 @@
+import { StepDescription, StepText } from './shared';
+import { OnboardingUseCase } from './types';
+import { OnboardingUseCasesTabsEnum } from './OnboardingUseCasesTabsEnum';
+import { useSegment } from '@novu/shared-web';
+import { CodeSnippet } from '../components/CodeSnippet';
+import { css } from '../../../styled-system/css';
+
+const COMMAND = 'npx novu-labs@latest echo';
+
+const EchoCodeSnippet = () => {
+  const segment = useSegment();
+
+  return (
+    <CodeSnippet
+      command={COMMAND}
+      // value from designs
+      className={css({ maxW: '400px' })}
+      onClick={() => {
+        segment.track(`Copy echo command - [Get Started]`);
+      }}
+    />
+  );
+};
+
+const textClassName = css({
+  color: 'legacy.B60',
```

Comment:
```suggestion
  color: 'typography.text.secondary',
```
We should try to only use semantic tokens in app code as much as possible